### PR TITLE
Add Qdrant module

### PR DESCRIPTION
**Pull request by Anush008**
This PR adds a module to spawn a [Qdrant](https://qdrant.tech) test container.

### CODE REVIEW

File: packages/modules/qdrant/src/qdrant-container.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+const QDRANT_REST_PORT = 6333;
+const QDRANT_GRPC_PORT = 6334;
+const QDRANT_CONFIG_FILE_PATH = "/qdrant/config/config.yaml";
+
+export class QdrantContainer extends GenericContainer {
+  private apiKey: string | undefined;
+  private configFilePath: string | undefined;
+
+  constructor(image = "qdrant/qdrant:latest") {
```
Comment (cristianrgreco) [inline packages/modules/qdrant/src/qdrant-container.ts:11]: Let's set a version explicitly, else one day a change will break existing builds

File: packages/modules/qdrant/src/qdrant-container.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    super(image);
+    this.withExposedPorts(QDRANT_REST_PORT, QDRANT_GRPC_PORT);
+    this.withWaitStrategy(Wait.forLogMessage(/Actix runtime found; starting in Actix runtime/));
+    this.withWaitStrategy(Wait.forHttp("/readyz", QDRANT_REST_PORT));
```
Comment (cristianrgreco) [inline packages/modules/qdrant/src/qdrant-container.ts:15]: `withWaitStrategy` will _replace_ an existing wait strategy. If you want multiple wait strategies you can use the CompositeWaitStategy, e.g:

```js
.withWaitStrategy(Wait.forAll([
  Wait.forLogMessage(/Actix runtime found; starting in Actix runtime/),
  Wait.forHttp("/readyz", QDRANT_REST_PORT)
]))
```

File: packages/modules/qdrant/tsconfig.build.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": [
+    "build",
+    "jest.config.ts",
+    "src/**/*.test.ts"
```
Comment (cristianrgreco) [inline packages/modules/qdrant/tsconfig.build.json:6]: If `test_config.yaml` is only used for testing we should add it here

File: packages/modules/qdrant/src/qdrant-container.ts (hunk 2 above)
Comment (Anush008) [reply packages/modules/qdrant/src/qdrant-container.ts]: https://github.com/testcontainers/testcontainers-node/pull/725/commits/723d9750c253f840ec536d81a7c2eeb0b8667505

**Comment by javierlopezdeancos** [comment]
I can't wait to use this one

**Review by javierlopezdeancos — APPROVED** [review]

**Pull request merged**